tests/e2e/test_rosacli_machine_pool.go

```diff
@@ -150,6 +152,35 @@ var _ = Describe("Create machinepool",
 				}
 			})
 
+		It("List instance-types with region flag - [id:72174]",
+			labels.Low, labels.Runtime.Day2,
+			func() {
+				By("List the available instance-types with the region flag")
+				typesList := []string{"dl1.24xlarge", "g4ad.16xlarge", "c5.xlarge"}
+				region := "us-east-1"
+				accountRolePrefix := fmt.Sprintf("QEAuto-accr72174-%s", time.Now().UTC().Format("20060102"))
+				_, err := ocmResourceService.CreateAccountRole("--mode", "auto",
+					"--prefix", accountRolePrefix,
+					"--permissions-boundary", permissionsBoundaryArn,
+					"-y")
+				Expect(err).To(BeNil())
+				accountRoleList, _, err := ocmResourceService.ListAccountRole()
+				Expect(err).To(BeNil())
+				classicInstallerRoleArn := accountRoleList.InstallerRole(accountRolePrefix, false).RoleArn
+				availableMachineTypes, _, err := ocmResourceService.ListInstanceTypesByRegion(
+					"--region", region, "--role-arn", classicInstallerRoleArn)
+
+				if err != nil {
+					log.Logger.Errorf("Failed to fetch instance types: %v", err)
```

Comment:
Shouldn't we use `Expect` and fail the test here ?

Reply:
Fixed